test/verify/check-storage-luks

```diff
@@ -289,7 +292,7 @@ class TestStorage(StorageCase):
         self.dialog_set_val("passphrase", "vainu-reku-toma-rolle-kaja-1")
         self.dialog_apply()
         b.wait_present(".dialog-error")
-        b.wait_in_text(".dialog-error", "Error unlocking /dev/sda: Failed to activate device: Operation not permitted")
+        self.assertIn(b.text(".dialog-error"), error_messages)
```

Comment:
I would have kept the wait_in_text with the first half, assert the second half, and drop the `error_base` bit completely. But waiting for presence should be fine, React shouldn't trigger a separate render for an empty error message.